### DISCUSSION TITLE
in-toto supply chain verification

### DISCUSSION THREAD

**Discussion by mattyn** [cdfoundation/sig-interoperability#74] [General]
As discussed in [Standardized Metadata](https://hackmd.io/BYbkuR8uSlKt_w7Y4KE1OQ), [in-toto](https://in-toto.io/in-toto/) is an existing effort to standardize some toolchain metadata.  In particular, it provides a standardized way for supply chain owners to define steps and verification instructions so that steps can be performed by various functionaries and their results can be verified independently by the supply chain owner and the consumer of the supply chain products. 

I'm starting this thread to gather ideas and reactions from the interoperability group about this standard and how it can contribute to CI/CD tool interoperability.

**Comment by fdegir** [comment]
Just an update - I had a chat with Santiago and he kindly accepted to join to our next SIG meeting on November 18th, 16:00UTC.
